mkdocs.yml

```diff
@@ -21,7 +21,18 @@ theme:
   favicon: https://koor.tech/images/favicon.svg
   logo: https://koor.tech/images/logo.svg
   palette:
-    scheme: "default"
+    - scheme: "default"
+      primary: "koor"
+      accent: "deep orange"
+      toggle:
+        icon: material/toggle-switch-off-outline
+        name: Switch to dark mode
+    - scheme: "slate"
+      primary: "koor"
+      accent: "red"
+      toggle:
+        icon: material/toggle-switch
+        name: Switch to light mode
```

Comment:
I like this! The top bar is still to bright though

![image](https://user-images.githubusercontent.com/3438324/209209149-29f1d61b-f3d2-4fab-851f-113399d8169e.png)

Reply:
Could you see if `#119671` as the `koor` color (https://github.com/koor-tech/koor/blob/feature/ksd-103/Documentation/stylesheets/extra.css#L11 and line 2) makes it less bright and still readable?

Fun fact the "green" color, Koor's primary logo color, is called "Medium Aquamarine".

Reply:
I'll add this as a new PR since it is out of the current scope.

Reply:
https://github.com/koor-tech/koor/pull/22

Reply:
I'll rebase this PR after your color changes have been merged.